ci/tests/vdisk/vdisk_test.py

```diff
@@ -89,3 +85,46 @@ def _get_scrubber_log_size():
                                    loop_device=loop)
 
         assert post_scrubber_logsize > pre_scrubber_logsize, "Scrubber actions where not logged!"
+
+    @staticmethod
+    def ovs_3756_metadata_size_test():
+        """
+        Validate get/set metadata cache size for a vdisk
+        """
+
+        metadata_cache_page_size = 256 * 24
+        default_metadata_cache_size = 8192 * metadata_cache_page_size
+
+        disk_name = 'ovs-3756-disk'
+        loop = 'loop0'
+        vpool = GeneralVPool.get_vpool_by_name(TestVDisk.vpool_name)
```

Comment:
if vpool is not found, nonetype error has no attribute storagedrivers will be thrown